packages/preact/hooks/src/useRange.ts

```diff
@@ -83,7 +87,7 @@ export function useRange(id: string) {
     return {
```

Comment:
move the whole return object to be a constant on module level instead of just FALLBACK_RANGE